sktm/jenkins.py

```diff
@@ -320,6 +326,51 @@ def build(self, baserepo=None, ref=None, baseconfig=None,
         logging.info("submitted build: %s", build)
         return build.get_number()
 
+    def __get_job(self):
+        """
+        Get Jenkens job by job name. Retry Jenkins $retry_cnt times in case
```

Comment:
This is a really minor nitpick, but can we do `self.retry_cnt` instead of the dollar notation? First, it's missing the `self` notion and `self.retry_cnt` and `retry_cnt` are two very different variables so someone new reading the code would get pretty confused; and secondly, well, it's python and not bash :)

Reply:
Well, now use self.retry_cnt instead of $retry_cnt, please check. Again, thank you very much for your review.